### ISSUE TITLE
#-1444240960------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1835]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<div class="sc-dxcDKg chkPkd">Running at 0 f/s</div>`

<strong>Element Location: </strong>.sc-dxcDKg

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\-1444240960_All page content should be contained by landmarks.png`
